.lintstagedrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "**/*.(j|t)s?(x)": (filenames) => [
+    `tsc-files --noEmit ${filenames.map((file) => "." + file.split(process.cwd())[1]).join(" ")}`,
+    `next lint --fix --file ${filenames.map((file) => file.split(process.cwd())[1]).join(" --file ")}`,
+  ],
+  "./prisma/schema.prisma": "prisma format",
+};
```

Comment:
why is it better to have this here?

Reply:
Next lint was having some issues with the package.json format